android/build.gradle

```diff
@@ -3,7 +3,15 @@
 buildscript {
     ext {
         buildToolsVersion = "29.0.2"
-        minSdkVersion = 16
+        minSdkVersion = 21 // see reason below as a suggestion from Android Studio
+        /**
+         * Manifest merger failed : uses-sdk:minSdkVersion 16 cannot be smaller than version 21 declared in library [:react-native-image-picker] /Users/brianmuks/projects/react-native/Expensify.cash/node_modules/react-native-image-picker/android/build/intermediates/library_manifest/debug/AndroidManifest.xml as the library might be using APIs not available in 16
+         * 	Suggestion: use a compatible library with a minSdk of at most 16,
+         * 		or increase this project's minSdk version to at least 21,
+         * 		or use tools:overrideLibrary="com.imagepicker" to force usage (may lead to runtime failures)
+         *
+         *
+         */
```

Comment:
I will need someone more familiar with Android to approve this change. cc @Julesssss @AndrewGable please advise. 

I think we don't need this comment if we are updating then it can be removed and the version change is ok.

Reply:
We should update`minimumSdkVersion` for the above mentioned warning, but I don't think this PR is the right place to do so. Let's keep the changes related to the current issue.

Reply:
@brianmuks are you blocked from compiling without this change? 'Manifest merger failed' make me believe this is the case, but that seems strange given that no-one else has been blocked by this.

@marcaaron tagged you on an internal issue.

Reply:
@Julesssss I couldn't build the App without making that change. 
FYI: I am using Android Studio 4.2 to build the app.

Reply:
I'd suggest building from the cli or using an older Android Studio version in that case. I'm able to build with Android Studio 4.0.1 currently.

Reply:
Noted with thanks.